src/components/grid/row.tsx

```diff
@@ -1,13 +1,37 @@
 import * as React from 'react';
-import css from 'classnames';
-import { row } from './style';
+import cx from 'classnames';
+import { Row as StyledRow } from './style';
 
-type Props = React.HTMLAttributes<HTMLDivElement>;
+export interface RowProps extends React.HTMLAttributes<HTMLDivElement> {
+  spacing?: number | string | null;
+}
 
-export const Row: React.FunctionComponent<Props> = ({ className, children, ...props }: Props): JSX.Element => {
+export const Row: React.FunctionComponent<RowProps> = ({
+  spacing,
+  className,
+  children,
+  ...props
+}: RowProps): JSX.Element => {
+  let formattedSpacingX;
```

Comment:
put spacing as default value here for both?

so you don't need to write ${formattedSpacingY || spacing}